app/controllers/renewals_controller.rb

```diff
@@ -37,7 +38,13 @@ def checkouts_to_renew
     end
 
     def renew_params
-      params.require(:renewal_list)
+      if params[:renewal_list].blank?
+        flash[:notice] = 'No items were selected for renewal.'
+
+        redirect_to checkouts_path
+      else
+        params[:renewal_list]
+      end
```

Comment:
I kinda feel like this should be a front-end validate, like unless a checkbox is checked don't allow the button to be clicked (i.e, disable it). What do you think? For the ruby code you can get rid of that `else`.

```suggestion
      end
      params[:renewal_list]